### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by tacaswell**
This replaces #2

rendered docs: http://tacaswell.github.io/docs/arch/arch.html

### CODE REVIEW

File: source/arch/begin_run_event.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
+        "beamline_config": {
+            "type": ["object", "string"],
+            "description": "Meta-data regrading how the beamline is configured.  Maybe UID into another collection"
+        },
+        "project": {
+            "type": "string",
```
Comment (ghost) [inline source/arch/begin_run_event.json:8]: How about we call this identifier? Project type might mean different things for different people.

Comment (danielballan) [inline source/arch/begin_run_event.json:8]: 'type' describes the allowed datatypes of the project field. There is no `project.type` field, just a `project` field that must be a string.

File: source/arch/begin_run_event.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
```
Comment (ghost) [inline source/arch/begin_run_event.json:2]: I am going to make "properties" an EmbeddedDocument. Any oppositions?

File: source/arch/begin_run_event.json (hunk 1 above)
Comment (ghost) [inline source/arch/begin_run_event.json:8]: Ahh got you!

File: source/arch/begin_run_event.json (hunk 2 above)
Comment (danielballan) [inline source/arch/begin_run_event.json:2]: I think you are misreading the schema. "Properties" is just a list of the fields; it is not something we actually implement.

Comment (ghost) [inline source/arch/begin_run_event.json:2]: Right! Ignore that comment. It was before we sorted out the project.type stuff below.

File: source/arch/ev_desc_prop.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                        "pattern": "^[A-Z]+:?",
+                        "type": "string",
+                        "description": "Where the data is stored if it is stored external to the events."
+                    },
+                    "shape": {
+                        "items": {
+                            "type": "integer"
+                        },
+                        "type": ["array", "null"],
+                        "destription": "The shape of the data.  Null and empty list mean scalar data."
```
Comment (ericdill) [inline source/arch/ev_desc_prop.json:28]: Typo (destription)

**Comment by danielballan** [comment]
This is not quite up to date, but it's a step in the right directly and it includes some mocking that will help me build the docs on my OSX laptop. So, merging.

**Pull request merged**